code/modules/mob/living/status_procs.dm

```diff
@@ -130,6 +130,7 @@
 	var/stunned = 0
 	var/stuttering = 0
 	var/weakened = 0
+	var/glowing = 0
```

Comment:
This file is intended for timed status effects, but it doesn't look like `glowing` is timed? Perhaps you could move this just to the `carbon_defines`, since the only thing that checks this variable is an organ and some species verbs.